lib/tasks/message_queue.rake

```diff
@@ -6,6 +6,16 @@
 require 'rummager'
 
 namespace :message_queue do
+  desc "Create queues"
```

Comment:
I'd suggest adding a bit more explanation here e.g. "Create the queues that Rummager uses with Rabbit MQ" as it doesn't really add a lot being same as the task name.